src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportErrorStore.cs

```diff
@@ -59,17 +60,24 @@ public async Task UploadErrorsAsync(string[] importErrors, CancellationToken can
                     await writer.WriteLineAsync(error);
                 }
 
+#pragma warning disable CA2016 // .NET 6 does not have cancel. Remove when we switch to .NET8
```

Comment:
I'd prefer the method level Suppression over this

Reply:
Removed